backend/src/oe/database.ts

```diff
@@ -52,6 +52,44 @@ export class DB {
       throw new Error( `${UUID} ERROR: with_accountPool: ${e instanceof Error? e.message: e}`);
     }
   }
+
+  private static async $blockSpanPool_getConnection(UUID: string): Promise<PrivatePoolConnection> {
+    logger.info( `${UUID} PROFILE: start: blockSpanPool.getConnection`);
+    const connection = await private_DB.blockSpanPool.getConnection();
+    private_DB.connections_count++;
+    logger.info( `${UUID} PROFILE: end: blockSpanPool.getConnection, current connections count ${private_DB.connections_count}`);
+    return ({ value: connection} as PrivatePoolConnection);
+  }
+  
+  public static async $blockSpanPool_query<T extends RowDataPacket[][] | RowDataPacket[] | OkPacket | OkPacket[] | ResultSetHeader>( UUID: string, connection: PrivatePoolConnection, query: string, args: any | any[] | { [param: string]: any }): Promise<[T, FieldPacket[]]> {
```

Comment:
Why are we using var? instead of let and const? any specific reason for it?

Reply:
you are correct here: const should be enough for this

Reply:
@dambaev I suggest adding the last optional parameter label with the default value accountPool. I did not suggest modifying accountPool_query method which is used at more than 20 places. It is fine we can rename it profile_query but did not suggest changing the arguments it may create some bugs if we did any mistake